Optimizers/optimizer_mppi_tf.py

```diff
@@ -73,6 +67,15 @@ def __init__(
             self.mppi_output = self.return_restricted
         
         self.optimizer_reset()
+    
+    def configure(self, dt: float, predictor_specification: str):
+        self.SQRTRHODTINV = tf.convert_to_tensor(np.array(self._SQRTRHOINV) * (1 / np.sqrt(dt)), dtype=tf.float32)
+        del self._SQRTRHOINV
+        
+        self.predictor_single_trajectory.configure(
+            batch_size=1, horizon=self.mpc_horizon, dt=dt,  # TF requires constant batch size
+            predictor_specification=predictor_specification,
+        )
```

Comment:
Example for an optimizer configuration: The `rho` parameter of MPPI depends on `dt`, which comes from the control frequency. It should be passed into the optimizer from its parent (the MPC controller), instead of hardcoding in config.